src/Components/Facility/FacilityCard.tsx

```diff
@@ -157,109 +157,107 @@ export const FacilityCard = (props: { facility: any; userType: any }) => {
                 </div>
               </div>
               <div className="flex flex-wrap border-t bg-gray-50 px-2 py-1 md:px-3">
-                <div className="flex justify-between py-2">
-                  <div className="flex w-full flex-wrap justify-between gap-2">
-                    <div className="flex flex-wrap gap-2">
-                      <div
-                        className={`tooltip ml-auto flex h-[38px] w-fit items-center justify-center rounded-md px-2 text-xl ${
+                <div className="flex w-full flex-wrap justify-between gap-2">
+                  <div className="flex flex-wrap gap-2">
+                    <div
+                      className={`tooltip ml-auto flex h-[38px] w-fit items-center justify-center rounded-md px-2 text-xl ${
+                        facility.patient_count / facility.bed_count > 0.85
+                          ? "button-danger-border bg-red-500"
+                          : "button-primary-border bg-primary-100"
+                      }`}
+                    >
+                      <span className="tooltip-text tooltip-bottom md:tooltip-right -translate-y-2">
+                        Live Patients / Total beds
+                      </span>{" "}
+                      <CareIcon
+                        className={classNames(
+                          "care-l-bed mr-2",
+                          facility.patient_count / facility.bed_count > 0.85
+                            ? "text-white"
+                            : "text-primary-600"
+                        )}
+                      />{" "}
+                      <dt
+                        className={`my-1 text-sm font-semibold ${
                           facility.patient_count / facility.bed_count > 0.85
-                            ? "button-danger-border bg-red-500"
-                            : "button-primary-border bg-primary-100"
+                            ? "text-white"
+                            : "text-gray-700"
```

Comment:
Why was this changed? This was intentionally kept to indicate almost full utilisation of a facility.